src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -920,6 +920,7 @@ export default class ChargingStationService {
     // Check and Get Charging Station
     const chargingStation = await UtilsService.checkAndGetChargingStationAuthorization(
       req.tenant, req.user, filteredRequest.ID, action, null, {
+        withSite: true,
```

Comment:
Good catch.
The migration has not yet been done on this endpoint but I would like that you pass all the params withXXX from the front-end, filter them with the Json Schema and enhance the OAS file (check also with Alix to perform this change also).

<img width="1009" alt="2021-09-08_16-28-18" src="https://user-images.githubusercontent.com/25266945/132528837-20c102b5-483f-4ec4-a63c-5f048f5d51c8.png">

Reply:
I changed this part but the other one you coded for the export OCPP params from Sites UI is still hardcoded, want to change?